roles/build_push_container/tasks/cleanup.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Remove registry
+  ansible.builtin.command:
+    cmd: podman "{{ item }}" registry
+  loop:
+    - stop
+    - rm
+
+- name: List created images
+  ansible.builtin.command:
+    cmd: podman images --filter=label=build_push_role=true --format "table {{ '{{' }}.ID{{ '}}' }}" --noheading
+  register: podman_images_output
+
+- name: Remove created images
+  ansible.builtin.command:
+    cmd: podman rmi -f {{ item }}
```

Comment:
note that `podman rmi -af` would remove *all* images.

Reply:
I would like this role to be generic enough to be called outside of my use case. In that case I wouldn't want all of someones local images being deleted if they ran the cleanup.

Reply:
Makes sense.